api/src/pcapi/routes/public/individual_offers/v1/endpoints.py

```diff
@@ -370,3 +364,46 @@ def get_event(event_id: int) -> serialization.EventOfferResponse:
         raise api_errors.ApiErrors({"event_id": ["The event offer could not be found"]}, status_code=404)
 
     return serialization.EventOfferResponse.build_event_offer(offer)
+
+
+@blueprint.v1_blueprint.route("/events/<int:event_id>/dates", methods=["GET"])
+@spectree_serialize(api=blueprint.v1_schema, tags=[EVENT_OFFERS_TAG], response_model=serialization.GetDatesResponse)
+@api_key_required
+def get_event_dates(event_id: int, query: serialization.GetDatesQueryParams) -> serialization.GetDatesResponse:
+    """
+    Get the dates of an event. Results are paginated.
```

Comment:
(détail, ou passe globale plus tard)On met beaucoup de "The" un peu inutile j'ai l'impression. `Get dates of an event.`
The maximum number of items per page. -> Maximum number of items per page